src/owncloud/autoconfig.php

```diff
@@ -3,23 +3,26 @@
 $snap_name = getenv('SNAP_NAME');
 $snap_developer = getenv('SNAP_DEVELOPER');
 if ($snap_developer == '') {
-        $snap_developer = getenv('SNAP_ORIGIN');
+	$snap_developer = getenv('SNAP_ORIGIN');
 }
 
 $snap_fullname = $snap_name . '.' . $snap_developer;
 
-$data_path = '/var/lib/snaps/'.$snap_fullname.'/current';
+$data_path = '/var/lib/snaps/' . $snap_fullname . '/current';
 
-$AUTOCONFIG = array(
-'directory' => $data_path.'/owncloud/data',
+$writable = getenv('SNAP_DATA');
+$owncloud_password = trim(file_get_contents($writable . '/mysql/owncloud_password'));
```

Comment:
> I don't understand. It does use the file

It does now, but only to define the variable. We'll just need to make sure we document what we're doing that so it doesn't appear crufty is all :smiley: . The only other place to put it is the php-fpm startup script, and I don't really feel like it belongs there either.